docs/source/tutorials/configuration.rst

```diff
@@ -437,6 +437,7 @@ A list of all CLI flags
      --custom_folders      Use the folders defined in the [custom_folders] section of the config file to define the output folder structure
      -v {DEBUG,INFO,WARNING,ERROR,CRITICAL}, --verbosity {DEBUG,INFO,WARNING,ERROR,CRITICAL}
                            Verbosity level. Also changes the ffmpeg log level unless the latter is specified in the config
+     --version VERSION     Print the current version of Manim you are using
```

Comment:
No argument should be passed here.

Reply:
Isn't the point of listing the CLI flags to show what can be run? While it wouldn't be passed as a flag when rendering, it is still a command.

Reply:
I think @eulertour means this line:
`      --version VERSION     Print the current version of Manim you are using `
should become this:
`      --version                    Print the current version of Manim you are using `